LibraryManagementSystem/Forms/ViewTab.cs

```diff
@@ -124,35 +124,35 @@ private void txtSearch_KeyDown(object sender, KeyEventArgs e)
         // Sort feature incomplete, currently in testing
         private void cmbSort_SelectedIndexChanged(object sender, EventArgs e)
         {
-            con = new SqlConnection("Data Source=" + Program.globalServer + "\\SQLEXPRESS;Initial Catalog=libraryData;Integrated Security=True");
+            con = new SqlConnection("Data Source=" + Program.globalServer + "\\SQLEXPRESS;Initial Catalog=LibDat;Integrated Security=True");
             con.Open();
             string cmdText;
 
             switch (cmbSort.SelectedIndex)
             {
                 // Sort Title in alphabetical order
                 case 0:
-                    cmdText = "SELECT * FROM booksData ORDER BY Title";
+                    cmdText = "SELECT * FROM book ORDER BY title";
                     cmd = new SqlCommand(cmdText, con);
                     break;
                 // Sort Author in alphabetical order
                 case 1:
-                    cmdText = "SELECT * FROM booksData ORDER BY Author";
+                    cmdText = "SELECT * FROM book ORDER BY author";
                     cmd = new SqlCommand(cmdText, con);
                     break;
                 // Sort BookID in ascending order
                 case 2:
-                    cmdText = "SELECT * FROM booksData ORDER BY BookID";
+                    cmdText = "SELECT * FROM book ORDER BY book_id";
                     cmd = new SqlCommand(cmdText, con);
                     break;
                 // Sort books by the date they were borrowed
                 case 3:
-                    cmdText = "SELECT * FROM booksData ORDER BY DateBorrowed";
+                    cmdText = "SELECT * FROM borrow_data ORDER BY borrow_date";
```

Comment:
Needs to be updated from <kbd>cmdText = "SELECT * FROM borrow_data ORDER BY borrow_date";</kbd> to something like the code in method *ViewTab_Load()* on line 34:
<kbd>cmdText = "SELECT book.book_id, book.title, book.author, book.genre, book.edition, book.status, bd.borrow_date, bd.return_date FROM book INNER JOIN borrow_data bd ON book.book_id = bd.book_id; ", con);</kbd>
While finding a way to add this to the SQL query: <kbd>ORDER BY borrow_date</kbd>